erpnext/selling/doctype/quotation/quotation.py

```diff
@@ -13,9 +13,14 @@
 }
 
 class Quotation(SellingController):
+	def after_insert(self):
+		self.update_lead()
+
 	def validate(self):
 		super(Quotation, self).validate()
 		self.set_status()
+		self.update_opportunity()
```

Comment:
Status of Lead or Opportunity should not be updated from a draft quotation, it should be only updated on submission of Quotation.